components/automate-ui/README.md

```diff
@@ -82,40 +82,6 @@ Reason: Later releases are release candidates; should only be using production-r
 
 Per https://github.com/chef/automate/pull/1867, future versions have made a breaking change so it breaks our code base.
 
-### Package tslint: ^5.20.1
-
-Reason: Trying to install the latest version `npm install tslint@6` reports these warnings
-so maintaining at highest version of 5.x.x:
-
-```text
-npm WARN deprecated tslint@6.0.0: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.
-npm WARN codelyzer@5.2.1 requires a peer of tslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
-npm WARN rxjs-tslint@0.1.7 requires a peer of tslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
-npm WARN tslint-defocus@2.0.6 requires a peer of tslint@^5.x but none is installed. You must install peer dependencies yourself.
-```
-
-### Package typescript: ~3.5.3
-
-Reason: Trying to install the latest version `npm install typescript@3.7.4` reports these warnings
-so maintaining at highest version of 3.5.x:
-
-```text
-@angular-devkit/build-angular@0.803.21 requires a peer of typescript@>=3.1 < 3.6 but none is installed.
-@angular/compiler-cli@8.2.14 requires a peer of typescript@>=3.4 <3.6 but none is installed.
-@ngtools/webpack@8.3.21 requires a peer of typescript@>=3.4 < 3.6 but none is installed.
-```
-
-### Package @nguniversal/express-engine: 9.0.0-rc.2
-
-Reason: `ng build` reported this error:
-
-```text
-ERROR in The target entry-point "ngx-cookie" has missing dependencies:
- - express-serve-static-core
-```
-
-Reference: https://github.com/salemdar/ngx-cookie/issues/106
-
```

Comment:
I've still got 3 Low severity warnings on Dev packages related to TSLint.  I think this is because as of 2019 TSLint is slowly being migrated to ESLint.  

These are Dev package warnings that won't make their way into production, so i'm not concerned about them right now, but this is likely something we should begin to address - migrating over to ESLint --> https://github.com/palantir/tslint/issues/4534

Reply:
The way forward--should we choose to do that--is well-documented here: https://dev.to/viveknayyar/fixing-security-vulnerabilities-in-npm-dependencies-in-less-than-3-mins-2p5g
However, if you "follow the money" and look further at [NPM Force Resolutions](https://www.npmjs.com/package/npm-force-resolutions), it is better to file an issue at the source rather than band-aid it on our end. 
Since `npm audit` reports these three are all low severity, and since our angular 11 update is around the corner, ignoring these for now seems most appropriate.

Reply:
Wonderful - I actually came across that article in my research but `force resolutions` scared me away 😂 .  I'll leave for now and will revisit in the 11 upgrade